manipulate_pim_data/connect/configure_event_subscription_network.rst

```diff
@@ -0,0 +1,16 @@
+How to Configure the Event Subscription network restrictions
+============================================================
+
+In the Event Subscription, some restrictions exists on the entered URL to guarantee that the PIM is not used for accessing
+unsolicited resources.
+However, **you may need to change those rules**, this chapter explains how.
```

Comment:
Should we need to explained that for security reasons some hostname are prohibited even if it present in the ALLOWED_NETWORK_WHITELIST ?

Reply:
you are right, `localhost` is a tricky one.
I don't know if the whitelist should supersede this particular rule ...